RFEM_Adapter/CRUD/Create/Panel.cs

```diff
@@ -61,19 +61,44 @@ private bool CreateCollection(IEnumerable<Panel> panels)
                     List<string> outlineNodeList = new List<string>();
                     foreach (Edge e in panelList[i].ExternalEdges)
                     {
+                        
                         //create rfem nodes, i.e. bhom points - NOTE: RFEM will remove the coincident points itself leaving jumps in node numbering ! 1,2,4,6,8,10,...
-                        Line edgeAsLine = e.Curve as Line;
 
-                        rf.Node rfNode1 = new rf.Node
+                        rf.Node rfNode1 = new rf.Node();
+
+                        if (e.Curve.GetType().Name.Equals("Polyline"))
```

Comment:
Think the above works, but you can simply do this instead, which imo reads a bit easier, and should run a bit faster.
```suggestion
                        if (e.Curve is Polyline)
```